app/src/main/res/values/strings.xml

```diff
@@ -70,6 +70,9 @@
   <string name="patiententry_error_phonenumber_length_less">Phone number cannot be less than 6 digits</string>
   <string name="patiententry_error_phonenumber_length_more">Phone number cannot be more than 12 digits</string>
   <string name="patiententry_identifier">BP Passport</string>
+  <string name="patiententry_consent_text"><b>Consent</b>: By tapping below, you confirm that any patient data you enter here has been obtained under appropriate informed consent. This means that the patient knows that you are entering their personal data into the app, understands what data is being collected, and knows that they may be contacted via SMS, WhatsApp, or other methods using the phone number provided in the app.
+\n\nAnyone authorized by you, or the healthcare provider for whom you work, will have access to the patient data and information you enter here. By using the app you confirm that the patient understands who will have access to the data obtained through this app and consents to that access.
```

Comment:
Let's not add formatting related information in `strings.xml`.
It might be difficult for this to work when translations happen.
One of the ways you can do this is split the string into three sections. 
1. Title `Consent`
2. First Paragraph
3. Second Paragraph

Reply:
Good point. Will discuss about this in person with you.

Reply:
Done